dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/query/DefaultQueryService.java

```diff
@@ -143,7 +149,7 @@ private long countObjects( Query query )
         return criteriaQueryEngine.count( pQuery );
     }
 
-    private List<? extends IdentifiableObject> queryObjects( Query query )
+    private List<? extends IdentifiableObject> queryObjects( Query query, boolean ignoreSharingSettings )
```

Comment:
Should we add `ignoreSharingSettings` as a property of `Query` object instead of new method parameter.

Reply:
Done